### PR TITLE
Externalize action config for the update plugin

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

/cc @kargakis @csrwng @paradigm

Externalize the prefix --> action. Normalize it to `make` target. Add response comment.

### CODE REVIEW

File: experiment/jenkins-config-updater/server.go [hunk 1]
```diff
@@ -45,6 +47,16 @@ type githubClient interface {
...
 }
 
+type ActionConfig struct {
+	Actions []Action `json:"actions"`
+}
+
+type Action struct {
+	Prefix     string `json:"prefix"`
+	MakeTarget string `json:"action"`
+	Name       string `json:"name"`
```
Comment (csrwng) [inline experiment/jenkins-config-updater/server.go]: is the MakeTarget not enough for the name?

Comment (stevekuznetsov) [reply experiment/jenkins-config-updater/server.go]: Hmmmm.... kind of wanted a displayname or something for humans

File: experiment/jenkins-config-updater/server.go [hunk 2]
```diff
@@ -152,17 +155,71 @@ func (s *Server) handleEvent(eventType, eventGUID string, payload []byte) error
...
+		commentBuffer.WriteString("The following configurations were updated:\n")
+	}
+	for _, task := range completedTasks {
+		commentBuffer.WriteString(fmt.Sprintf(" - %s\n", task))
+	}
 
-	if err != nil {
-		return err
+	if len(failedTasks) > 0 {
+		commentBuffer.WriteString("The following configurations were not updated due to failures:\n")
```
Comment (csrwng) [inline experiment/jenkins-config-updater/server.go]: nit: we won't know what configs were not updated, we only know that the task (that potentially updates many configs) succeeded or failed

File: experiment/jenkins-config-updater/server.go (hunk 1 above)
Comment (csrwng) [reply experiment/jenkins-config-updater/server.go]: ack

**Comment by csrwng** [comment]
a couple of comments, otherwise lgtm

**Comment by stevekuznetsov** [comment]
- <details>
  <summary>whatever</summary>
  test
  </details>
- <details>
  <summary>whatever</summary>
  test
  </details>
- <details>
  <summary>whatever</summary>
  test
  </details>

**Comment by stevekuznetsov** [comment]
Whoopsie

**Comment by stevekuznetsov** [comment]
@csrwng updated comment

**Comment by stevekuznetsov** [comment]
Hmm, will get rid of the human readable name, make target is good enough

**Pull request merged**